docs/messages.json

```diff
@@ -70,6 +70,63 @@
           "http://messages.beamos.mr-beam.org/img/Testimonial%20EN.png"
         ]
       }
+    },
+    {
+      "id": 3,
+      "date": "27. 07. 2022",
+      "restrictions": {
+        "__": "Evaluation  of restrictions needs to be true to show the message",
+        "channels": [
+          ""
+        ],
+        "version": [
+          "0.11.0rc1"
```

Comment:
this will not work, as it blacklists the named version, change to
version_and_newer: 0.11.0
version_and_older: 0.11.0b9
and test if this will work as expected, only show for version 0.11.0rc1